### PR TITLE
add --log_type flag to fulcio to enable production logging

### PR DESCRIPTION
**Pull request by priyawadhwa**
we need this to close out one of the remaining items in https://github.com/sigstore/public-good-instance/issues/387

Signed-off-by: Priya Wadhwa <priya@chainguard.dev>

## Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). Where applicable, update and bump the versions in any associated umbrella chart
- [ ] Variables are documented in the `values.yaml` and added to the README.md. The [helm-docs](https://github.com/norwoodj/helm-docs) utility can be used to generate the necessary content. Use `helm-docs --dry-run` to preview the content.
- [ ] JSON Schema generated.
- [ ] List tests pass for Chart using the [Chart Testing](https://github.com/helm/chart-testing) tool and the `ct lint` command.

### CODE REVIEW

File: charts/fulcio/templates/fulcio-deployment.yaml [hunk 1]
```diff
@@ -51,6 +51,9 @@ spec:
             - "--fileca-key-passwd"
             - "$(PASSWORD)"
             {{- end }}
+            {{- if .Values.server.logging.production }}
+            - "--log_type=prod"
```
Comment (k4leung4) [inline charts/fulcio/templates/fulcio-deployment.yaml:55]: should we pass the log_type rather than a boolean?
in the odd chance we need debugging logs?

**Review by k4leung4 — COMMENTED** [review]
please update README.md and schema

File: charts/fulcio/templates/fulcio-deployment.yaml (hunk 1 above)
Comment (priyawadhwa) [reply charts/fulcio/templates/fulcio-deployment.yaml:55]: there's only two types right now, dev (default) and prod. i copied this from the rekor chart from consistency, but can update both if you think that's better

Comment (k4leung4) [reply charts/fulcio/templates/fulcio-deployment.yaml:55]: dont feel strongly, fine as is for consistency

please update readme and schema with new value

**Comment by priyawadhwa** [comment]
updated, thanks for the reminder!

**Review by k4leung4 — APPROVED** [review]
thank you

**Pull request merged**